### PR TITLE
add fluid dist example

### PR DESCRIPTION
**Pull request by Yancey0623**

### CODE REVIEW

File: doc/boss_tutorial.md [hunk 1]
```diff
@@ -55,8 +55,62 @@ utilization of a cluster.
 
 ## Part-1 Training Models on Your Laptop using PaddlePaddle
 
-Please checkout [PaddlePaddle Book](http://github.com/PaddlePaddle/book), steps to run
-the training process and example output.
+### PaddlePaddle Book
+
+    Please checkout [PaddlePaddle Book](http://github.com/PaddlePaddle/book), steps to run
```
Comment (typhoonzero) [inline doc/boss_tutorial.md]: No need to indent here? markdown will handle the sections

File: doc/boss_tutorial.md [hunk 2]
```diff
@@ -55,8 +55,62 @@ utilization of a cluster.
...
+    the training process and example output.
+
+### Launch a Distributed Training Job on Your Laptop
+
+1. Launch the PaddlePaddle Production Docker Container:
+
+    ``` bash
+    > cd example/fluid
```
Comment (typhoonzero) [inline doc/boss_tutorial.md]: add `git clone` here? clone edl repo first

File: example/fluid/dist_word2vec.py [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+    act = sys.argv[1]
+    if act == 'prepare':
+        prepare_dataset()
+    elif act == "train":
+        main()
+    else:
+        print(usage)
+    
+
```
Comment (typhoonzero) [inline example/fluid/dist_word2vec.py]: extra empty line

**Review by typhoonzero — DISMISSED** [review]
LGTM just small comments

File: doc/boss_tutorial.md (hunk 1 above)
Comment (Yancey0623) [reply doc/boss_tutorial.md]: done.

File: doc/boss_tutorial.md (hunk 2 above)
Comment (Yancey0623) [reply doc/boss_tutorial.md]: done.

File: example/fluid/dist_word2vec.py (hunk 3 above)
Comment (Yancey0623) [reply example/fluid/dist_word2vec.py]: done.

Comment (Yancey0623) [reply example/fluid/dist_word2vec.py]: done.

Comment (Yancey0623) [reply example/fluid/dist_word2vec.py]: done.

Comment (Yancey0623) [reply example/fluid/dist_word2vec.py]: done.

**Review by typhoonzero — APPROVED** [review]
LGTM++

**Pull request merged**